docs/configuring-playbook-bridge-mautrix-telegram.md

```diff
@@ -0,0 +1,5 @@
+# Setting up Mautrix Telegram
+
+The playbook can install and configure [mautrix-telegram](https://github.com/tulir/mautrix-telegram) for you.
+
+See that project's [documentation](https://github.com/tulir/mautrix-telegram/wiki#usage) to learn what it does and why it might be useful to you.
```

Comment:
That's nice!

Maybe it can also have some snippet with what variables need to be defined to get the playbook to install the bridge?

Like:

> To get it installed, you'd need to obtain API keys from https://my.telegram.org/apps and then use the following playbook configuration:
> 
> ```yaml
> matrix_mautrix_telegram_enabled: true
> matrix_mautrix_telegram_api_id: YOUR_TELEGRAM_APP_ID
> matrix_mautrix_telegram_api_hash: YOUR_TELEGRAM_API_HASH
> ```